elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -80,6 +80,12 @@ class PfeContentSet extends PFElement {
       oldId: {
         alias: "id",
         attr: "pfe-id"
+      },
+      upgraded: {
```

Comment:
What's this used for?  Asking because `this.isConnected` is part of the web component spec and could probably be used instead.

Reply:
This was going to be how we tag light DOM elements as already having been migrated.  The mutation observer would remove it until the element is moved to the shadow DOM